salt/annotations/config/home-deployuser-annotations-docker-compose.yml

```diff
@@ -12,4 +12,14 @@ services:
         ports:
             - 9000:9000
         restart: always
+    queue_watch:
+        image: "elifesciences/annotations_cli:${IMAGE_TAG}"
```

Comment:
create a container from the image with this tag, mounting configuration, log folder and AWS credentials.

Provide environment variables (only the environment name for the console). Restart on reboot.